content/en/guide/v10/no-build-workflows.md

```diff
@@ -0,0 +1,123 @@
+---
+name: No-Build Workflows
```

Comment:
Switched this over to 'No-Build Workflows' instead, as an importmap-specific page felt unnecessary. Granted, this might be a bit confusing for Deno users, but I felt that the ability to more info here in the future for buildless workflows is of more benefit.

Will think on it, anyhow. Opinions certainly welcome (as always).